cmd/describe/cluster/cmd.go

```diff
@@ -102,6 +102,27 @@ func run(cmd *cobra.Command, argv []string) {
 	var upgradeState *cmv1.UpgradePolicyState
 	var controlPlaneScheduledUpgrade *cmv1.ControlPlaneUpgradePolicy
 
+	// SDN -> OVN Migration Status
+	migrationsStr := ""
+	collection, err := r.OCMClient.FetchClusterMigrations(cluster.ID())
+	migrations := map[string]*cmv1.ClusterMigration{}
+	if err != nil {
+		if !strings.Contains(err.Error(), "is not enabled for this organization") {
+			r.Reporter.Warnf("Failed to get cluster migrations: %v", err)
+		}
+	} else if len(collection.Items().Items()) > 0 {
```

Comment:
Intentionally nested if statement, so that the other branch does not run if the error is a feature toggle error.

Reply:
nit: Do you need the else?